Week-01/Day-02_Finding-Nemo/finding_nemo_botiboti/src/lib.rs

```diff
@@ -0,0 +1,39 @@
+#[allow(non_snake_case)]
+
+pub fn findNemo(inp: &str) -> String {
+    match inp.split_whitespace().position(|x| x == "Nemo") {
+        Some(index) => "I found Nemo at ".to_string() + &(index + 1).to_string() + "!",
+        None => "I can't find Nemo :(".to_string(),
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn no_nemo() {
```

Comment:
Nice tests!
Please add some more for the following cases:
- Nemo in the middle of the string
- Nemo at the end of the string
- Nemo's